packages/server/lib/user.ts

```diff
@@ -1,69 +0,0 @@
-const debug = require('debug')('cypress:server:user')
-const api = require('./api')
-const cache = require('./cache')
-const errors = require('./errors')
-
-export = {
-  get () {
-    return cache.getUser()
-  },
-
-  set (user) {
-    return cache.setUser(user)
-  },
-
-  getBaseLoginUrl () {
-    return api.getAuthUrls()
-    .get('dashboardAuthUrl')
-  },
-
-  logOut () {
-    return this.get().then((user) => {
-      const authToken = user && user.authToken
-
-      return cache.removeUser().then(() => {
-        if (authToken) {
-          return api.postLogout(authToken)
-        }
-      })
-    })
-  },
-
-  syncProfile (authToken) {
-    debug('synchronizing user profile')
-
-    return api.getMe(authToken)
-    .then((res) => {
-      debug('received /me %o', res)
-      const user = {
-        authToken,
-        name: res.name,
-        email: res.email,
-      }
-
-      return this.set(user)
-      .return(user)
-    })
-  },
-
-  ensureAuthToken () {
```

Comment:
This was replaced by https://github.com/cypress-io/cypress/blob/develop/packages/data-context/src/actions/AuthActions.ts#L35